### ISSUE TITLE
second curly bracket is removed when block is used as a param

### ISSUE THREAD

**Issue by yratanov** [AaronC81/sord#136]
**To Reproduce**
```rb
class A
  def x(a: ->() {}); end
end
```

**Expected behavior**
```rb
class A
  # sord omit - no YARD type given for "a:", using untyped
  # sord omit - no YARD return type given, using untyped
  sig { params(a: T.untyped).returns(T.untyped) }
  def x(a: ->() {}); end
end
```

**Actual behavior**

```rb
class A
  # sord omit - no YARD type given for "a:", using untyped
  # sord omit - no YARD return type given, using untyped
  sig { params(a: T.untyped).returns(T.untyped) }
  def x(a: ->() {); end
end
```

```diff
- def x(a: ->() {); end
+ def x(a: ->() {}); end
```

Failing test: https://github.com/AaronC81/sord/pull/137

**Comment by AaronC81** [comment]
I reckon this is because of the workaround in #127 to fix a YARD bug. The default parameter in your example kind of begins with a unary minus symbol, but it's actually a `->`.

I'll take a look at this in the next couple of days! It don't think it should be too tricky to exclude this case from that workaround.

**Comment by AaronC81** [comment]
Just tidying up - fixed by #137, so closing :)
